src/tracking.js

```diff
@@ -114,7 +114,7 @@ export function setupLogger() {
     // https://www.paypal.com/us/gifts/
     const isLoadedInFrame = isPayPalDomain() && window.xprops;
     const sdkLoadTime =
-      typeof loadTime === "number" ? loadTime.toString() : undefined;
+      typeof loadTime === "number" ? Number(loadTime) : undefined;
```

Comment:
isn't loadTime already a number in this case? since `typeof loadtime === "numer"` is true?